### PR TITLE
Support redirects outside of the standard render flow

### PR DESCRIPTION
**Pull request by ganemone**

### CODE REVIEW

File: src/browser.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ export default __BROWSER__ &&
     middleware: () => {
       return (ctx, next) => {
         ctx.element = (
-          <HelmetProvider content={{}}>{ctx.element}</HelmetProvider>
+          <HelmetProvider context={{}}>{ctx.element}</HelmetProvider>
```
Comment (KevinGrandon) [inline src/browser.js:10]: Feels weird that we pass in an empty object?

**Review by KevinGrandon — DISMISSED** [review]

File: src/browser.js (hunk 1 above)
Comment (ganemone) [reply src/browser.js:10]: Afaict that is what we are supposed to do.

**Review by KevinGrandon — APPROVED** [review]

**Comment by ganemone** [comment]
!merge

File: src/__tests__/request.node.js [hunk 2]
```diff
@@ -11,3 +11,14 @@ test('Non render request', async t => {
   t.equal(ctx.element, null, 'does not wrap ctx.element');
   t.end();
 });
+
+test('Render request with server side redirect', async t => {
```
Comment (mlmorg) [inline src/__tests__/request.node.js:15]: It's unclear to me what this test is actually testing. There are no assertions.

**Review by mlmorg — COMMENTED** [review]
No description? What is this for? Test is also unclear.

File: src/__tests__/request.node.js (hunk 2 above)
Comment (ganemone) [reply src/__tests__/request.node.js:15]: It is testing that the app can go render with a server side redirect

Comment (mlmorg) [reply src/__tests__/request.node.js:15]: Should it assert that the page is now `/test`? Or am I confusing what this is actually doing?

**Pull request merged**